openpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -127,9 +127,9 @@ def process(self, instance):
             output_frame_start
         )
 
-        # Fill tags and new families
+        # Fill tags and new families from project settings
         tags = []
-        if family_lowered in ("review", "renderlayer", "renderscene"):
+        if family_lowered in self.families_to_upload:
```

Comment:
Those are not families to upload but families to add review tag.
```suggestion
        if family_lowered in self.families_to_review:
```

Also please define `families_to_review` as class attribute of the plugin with default value.